src/main/java/org/traccar/protocol/NavtelecomProtocolDecoder.java

```diff
@@ -293,9 +310,90 @@ protected Object decode(
                                 case 51:
                                 case 52:
                                     value = buf.readByte();
-                                    position.set(
-                                            Position.PREFIX_TEMP + (j + 2 - 45),
-                                            (value != (byte) 0x80) ? value : null);
+                                    if (value != (byte) 0x80) {
+                                        position.set(Position.PREFIX_TEMP + (j + 2 - 45), value);
+                                    }
+                                    break;
+                                case 53:
+                                    value = buf.readUnsignedShortLE();
+                                    if (value != 0x7FFF) {
+                                        if (BitUtil.check(value, 7)) {
+                                            position.set("fuelLevel", BitUtil.to(value, 6));
+                                        } else {
+                                            position.set(Position.KEY_FUEL_LEVEL, BitUtil.to(value, 6) / 10);
```

Comment:
Please explain

Reply:
This is still not addressed. What's the difference between fuelLevel and KEY_FUEL_LEVEL?